src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1084,11 +1080,11 @@ private void startOauthorization() {
     }
 
     private void openUrlWithCustomTab(String url) {
-        TypedValue tvBackground = new TypedValue();
-        getTheme().resolveAttribute(R.attr.colorPrimary, tvBackground, true);
+        TypedValue backgroundCollor = new TypedValue();

```

Comment:
backgroundColor instead of backgroundCollor

Reply:
done